app.json

```diff
@@ -139,6 +146,271 @@
+				"nl": "TSP01 3 in 1 PIR Motion Sensor"
+			},
+			"zwave": {
+				"manufacturerId": 280,
+				"productTypeId": 2,
+				"productId": [
+					2
+				],
+				"learnmode": {
+					"image": "/drivers/TSP01/assets/learnmode.svg",
+					"instruction": {
+						"en": "Press button 3 times within 1.5 seconds",
+						"nl": "Druk 3 keer op de knop binnen 1,5 seconden"
+					}
+				},
+				"associationGroups": [
+					1,
+					2
+				],
+				"associationGroupsOptions": {
+					"1": {
+						"hint": {
+							"en": "Report message group"
+						}
+					},
+					"2": {
+						"hint": {
+							"en": "Light control group"
+						}
+					}
+				}
+			},
+			"class": "sensor",
+			"capabilities": [
+				"alarm_motion",
+				"alarm_tamper",
+				"measure_luminance",
+				"measure_temperature",
+				"measure_battery"
+			],
+			"images": {
+				"large": "/drivers/TSP01/assets/images/large.png",
+				"small": "/drivers/TSP01/assets/images/small.png"
+			},
+			"settings": [
+				{
+					"type": "group",
+					"label": {
+						"en": "Homey driver settings",
+						"nl": "Homey driver configuratie"
+					},
+					"children": [
+						{
+							"id": "motion_inactivity_timer",
+							"type": "number",
+							"attr": {
+								"min": 0,
+								"max": 86400
+							},
+							"value": 30,
+							"label": {
+								"en": "Manual motion alarm inactivity timer",
+								"nl": "Bewegingssensor timeout (handmatig)"
+							},
+							"hint": {
+								"en": "Determine the amount of seconds after which homey decides to reset the motion sensor to OFF after no motion detected. Range: 0 (Off), 30 - 86400 seconds",
+								"nl": "Bepaal het aantal seconden waarna homey de status van de bewegingssensor reset naar UIT als er geen beweging meer is gedetecteerd. Bereik: 0 (Off), 30 - 86400 seconden"
+							}
+						},
+						{
+							"id": "motion_interact_class_basic",
+							"type": "checkbox",
+							"value": false,
+							"label": {
+								"en": "Motion alarm act on lights ON/OFF command from device",
+								"nl": "Bewegingssensor aan/uitzetten via licht aan/uit commando van apparaat"
+							},
+							"hint": {
+								"en": "Allow Homey to use the CLASS_BASIC_SET report to en-/disable the motion sensor status",
+								"nl": "Laat Homey de motion sensor status aan-/uitzetten n.a.v. het CLASS_BASIC_SET rapport (licht aan/uit status vanuit motion detector)."
+							}
+						}
+					]
+				},
+				{
+					"type": "group",
+					"label": {
+						"en": "Z-Wave configuration",
+						"nl": "Z-Wave configuratie"
+					},
+					"children": [
+						{
+							"id": "basic_set_level",
+							"type": "number",
+							"attr": {
+								"min": -1,
+								"max": 100
+							},
+							"label": {
+								"en": "Basic Set Level",
+								"nl": "Basic Set waarde"
+							},
+							"hint": {
+								"en": "Setting the BASIC command value to turn on the light. The 0xFF (-1) means turn on the light.\nFor dimmer equipment 1 to 100 means the light strength.",
```

Comment:
During dev i first tried the 255 value, but that did not work out... (did not get the 0xFF value, but was capped at 127, being a signed int, resulting in 0x7F or something)